react/src/paybutton-config.json

```diff
@@ -1,4 +1,9 @@
 {
   "wsBaseUrl": "https://socket.paybutton.org",
-  "apiBaseUrl": "https://paybutton.org"
+  "apiBaseUrl": "https://paybutton.org",
+  "defaultCurrency": "xec",
```

Comment:
Since this is very specific to the storybook, I would make that clear in the name. Otherwise it can be confusing as if it was a default address/currency for the paybutton component. Something like `defaultStorybookCurrency`.

Reply:
That sounds good to me -- PayButton auto-detects the currency based on the address we pass into it.

Reply:
If it is only the storybook, I don't mind changing the var name later if we need to use it, specially in this case since it would be an easy name to do search-and-replace.

But indeed I see that the defaultCurrency is being used on the Widget component, regardless of the storybook. So I think it makes sense to have defaultCurrency, but the address I think should be defaultStorybookAddress (no need to have Wallet in the name, since we don't use Wallet nomenclature in this repo and in the server it means a different thing).

Reply:
> If it is only the storybook, I don't mind changing the var name later if we need to use it, specially in this case since it would be an easy name to do search-and-replace.
> 
> But indeed I see that the defaultCurrency is being used on the Widget component, regardless of the storybook. So I think it makes sense to have defaultCurrency, but the address I think should be defaultStorybookAddress (no need to have Wallet in the name, since we don't use Wallet nomenclature in this repo and in the server it means a different thing).

I just changed the logic to be inside the `.storybook` since it will be used only by storybook for testing and showcase 
Also implemented it in a way we calculate the current currency by the crypto address